### PR TITLE
fix error

### PR DESCRIPTION
**Pull request by jvmsangkal**

### CODE REVIEW

**Comment by ravenjohn** [comment]
<a href="http://ninja.raven.tm/anyTV/music-dashboard/pull/394" target="_blank">ReviewNinja

**Comment by jvmsangkal** [comment]
:AllenApproves:

File: app/scripts/components/common/upload/bulk/index.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ var React = require('react'),
                 yt_upload = +this.refs['cbYoutube'].getDOMNode().checked,
                 data = {
                     folder: this.refs.track0.getDOMNode().files[0],
-                    yt_upload,
+                    yt_upload: yt_upload,
```
Comment (ravenjohn) [inline app/scripts/components/common/upload/bulk/index.js]: is this really needed?

Comment (ravenjohn) [inline app/scripts/components/common/upload/bulk/index.js]: ohhh. this is client side? then this should be fine

Comment (jvmsangkal) [inline app/scripts/components/common/upload/bulk/index.js]: yes

**Comment by ravenjohn** [comment]
:AllenApproves:

**Pull request merged**